main.py

```diff
@@ -2,8 +2,9 @@
 from tkinter import filedialog
 
 class Application(Frame):
-    def __init__(self, master=None):
+    def __init__(self, master=None, defaultPath="C:/"):
```

Comment:
defaultPath was just an example. There is no need to send the path as a parameter from main app to your class.
The path must be selected only withing the app